api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -1151,6 +1151,32 @@ message ListRolesResponse {
 
   // NextKey will serve as the StartKey for the next page of roles.
   string NextKey = 2;
+
+  // RequestableRoles is a page of requestable roles. This will be populated if the request only requested requestable roles.
+  repeated types.RequestableRole RequestableRoles = 3;
+}
+
+// ListRequestableRolesRequest is the request to list requestable roles.
+message ListRequestableRolesRequest {
+  // Limit is the maximum amount of roles per page.
+  int32 Limit = 1;
+
+  // StartKey is used to resume a query in order to enable pagination.
+  // If the previous response had LastKey set then this should be
+  // set to its value. Otherwise leave empty.
+  string StartKey = 2;
```

Comment:
nit: I think we're trying to standardize on `page_token`/`next_page_token`.  Idk about wether or not we're doing snake case in legacy, but having this at least be `PageToken`/`NextPageToken` would probably be more consistent.